src/main/java/com/google/sps/servlets/CreateEventServlet.java

```diff
@@ -18,7 +18,7 @@
 /**
 * Servlet creates activity event, adds it to the user's calendar, and sends calendar invites to guests.
 */
-@WebServlet("/create-event")
+@WebServlet("/api/create-event")
```

Comment:
I think I added the api/ to the server addresses some time ago. Make sure to remove it because it's no longer compatible with current code.